src/ripple/core/tests/Config.test.cpp

```diff
@@ -346,10 +351,102 @@ port_wss_admin
                     "dbPath No Path");
         }
     }
+
+    void testSetup(bool explicitPath)
+    {
+        // TODO: This will conflict with PR 1702
```

Comment:
Well, it _will_ conflict. However, since #1702 is passed, I'm going to rebase off of it, and resolve the conflicts now, so the merge later will go more smoothly.